zeek/plugin/include/protocol-analyzer.h

```diff
@@ -2,18 +2,17 @@
 
 #pragma once
 
+#include "cookie.h"
```

Comment:
Any reason we include this differently than `zeek-sicy/zeek-compat.h` below? I'd suggest to stick to a single style, e.g.,
```suggestion
#include <zeek-spicy/cookie.h>
```

Reply:
Ack, fixing.